docs/compose-file.md

```diff
@@ -552,7 +552,7 @@ limit as an integer or soft/hard limits as a mapping.
         soft: 20000
         hard: 40000
 
-### volumes, volume\_driver
+### volumes
 
 Mount paths or named volumes, optionally specifying a path on the host machine
 (`HOST:CONTAINER`), or an access mode (`HOST:CONTAINER:ro`). Named volumes can
```

Comment:
Here's what I have:

> Mount paths or named volumes, optionally specifying a path on the host machine
(`HOST:CONTAINER`), or an access mode (`HOST:CONTAINER:ro`).

> With the [version 2 file format](#version-2), named volumes must be specified
with the [top-level `volumes` key](#volume-configuration-reference). If you're
using [version 1](#version-1), named volumes will be created implicitly when
declared in this section.

WDYT?